FindTarget.py

```diff
@@ -438,19 +438,23 @@ def findTape(contours, image, centerX, centerY, mask, CornerMethod, MergeVisionP
     screenHeight, screenWidth, channels = image.shape
     # Seen vision targets (correct angle, adjacent to each other)
     targets = []
+    # Constant used as minimum for fingerprinting equal to 75% of area of outer target at 30ft
+    minContourArea = 0.75 * 500;
 
     if len(contours) >= 1:
         # Sort contours by area size (biggest to smallest)
         cntsSorted = sorted(contours, key=lambda x: cv2.contourArea(x), reverse=True)[:10]
        
         cntsFiltered = []
 
-        if cntsSorted:
+        # First contour has largest area, so only go further if that one meets minimum area criterion
+        if cntsSorted and cv2.contourArea(cntsSorted[0]) > minContourArea:
```

Comment:
I don't understand why you would check only the first value here for Area?    Why not just check when the rest of the filters are checked?

Also, we  have a method called checkTargetSize() that used area.  It has been removed.   Why not just have a method called checkTargetFingerPrint() or something and do all the fingerprinting in one method?

Reply:
Another point is you are doing another call to cv2.contourArea() when it should only have to be called once.

Reply:
Updated based on our conversation. Please have a look at our changes